twentyFivePair_colour_code_config.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
```

Comment:
twentyFivePair is a number that is true as of now. But in case if you decide to add one more major color or minor color, then the naming will not match the content!
The naming of files can be independent of the number and can be improved upon.

Reply:
@prithamvshetty , should i work on to change the file name for this PR as a part of this assignment.? or is it just a suggestion/comment which i will take care in my future coding?

Reply:
Hi Ranjith, understanding the reason why it is needed is one thing and practicing it is another which is very important.  If you are able to address, it would be a good outcome 🙂.